circleci.sh

```diff
@@ -51,12 +51,18 @@ clone() {
 }
 
 coverage() {
-    if [ -n ${CIRCLE_PR_NUMBER:-} ]; then
-        local base_branch=$(curl -fsSL https://api.github.com/repos/dlang/dmd/pulls/$CIRCLE_PR_NUMBER | jq -r '.base.ref')
+    base_branch="master"
+    if [ -n "${CIRCLE_PR_NUMBER:-}" ]; then
+        base_branch=$(curl -fsSL https://api.github.com/repos/dlang/dmd/pulls/$CIRCLE_PR_NUMBER | jq -r '.base.ref')
```

Comment:
If curl fails, this will be set to an empty base branch.
Better check for empty string below `if [ -z $base_branch ]; then base_branch=master; fi`.